components/fms-consumer/src/main.rs

```diff
@@ -148,6 +195,17 @@ async fn process_protobuf_message(
     }
 }
 
+async fn process_zenoh_message(
+    payload: &[u8],
+    influx_writer: Arc<InfluxWriter>,
+){
+              if let Some(vehicle_status) = deserialize_vehicle_status(payload) {
+                influx_writer.write_vehicle_status(&vehicle_status).await;
+            } else {
+        	debug!("ignoring message without payload");
+    	    }
+}
+
 async fn process_message(m: &BorrowedMessage<'_>, influx_writer: Arc<InfluxWriter>) {
```

Comment:
Since we support both Kafka and Zenoh in the program, the function name should be clear.

Reply:
function name updated to distinguish  zenoh and kafka.